### PR TITLE
Remove file level docstrings

### PR DESCRIPTION
**Pull request by mattdangerw**
Pet peeve. These are inconsistent, usually just a rephrase of the file itself, and keras-core isn't doing them.

Let's ditch filename docstring (unless they actually say something interesting). We can keep out class/function docstrings as the place to focus on.

### CODE REVIEW

File: keras_nlp/backend/config.py [hunk 1]
```diff
@@ -11,6 +11,7 @@
 # WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
 # See the License for the specific language governing permissions and
 # limitations under the License.
+
```
Comment (ianstenbit) [inline keras_nlp/backend/config.py:14]: Are all these new blank lines intentional? Version change in linter?

Comment (ianstenbit) [reply keras_nlp/backend/config.py:14]: Ooh this must be some find-replace shenanigans 😛

Comment (ianstenbit) [reply keras_nlp/backend/config.py:14]: Either way looks like it makes these files consistent with the rest so LGTM

**Review by ianstenbit — APPROVED** [review]

File: keras_nlp/backend/config.py (hunk 1 above)
Comment (mattdangerw) [reply keras_nlp/backend/config.py:14]: Yeah, we were not consistent, nor did our lint check. This was just a little script I wrote. No strong opinion on newline after copyright block or not.

**Pull request merged**